pkg/util/pki/certificatetemplate.go

```diff
@@ -213,6 +213,10 @@ func CertificateTemplateFromCSR(csr *x509.CertificateRequest, validatorMutators
 			template.UnknownExtKeyUsage = unknownUsages
 		}
 
+		if val.Id.Equal(oidExtensionSubjectAltName) {
+			template.ExtraExtensions = append(template.ExtraExtensions, val)
+		}
```

Comment:
I think this section has to be made optional, so issuers can decide themselves if they want to support otherName or not.

Reply:
I did rethink this and think this is OK, there will be very few cases where the SANs are being edited after the template was generated AFAIK.